### PR TITLE
update: readme, setup scripts, release scripts

### PR DESCRIPTION
**Pull request by zenixls2**
for fixedpoint updates.  
now releasing new version only requires updating tagname.sh

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -119,3 +119,75 @@ jobs:
...
           asset_name: bbgo-${{ steps.get_version.outputs.VERSION }}-darwin-arm64.tar.gz
           asset_content_type: application/octet-stream
+      - name: Upload slim Linux amd64 dnum
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: dist/${{ steps.get_version.outputs.VERSION }}/bbgo-slim-${{ steps.get_version.outputs.VERSION }}-linux-amd64-dnum.tar.gz
+          asset_name: bbgo-slim-${{ steps.get_version.outputs.VERSION }}-linux-amd64-dnum.tar.gz
```
Comment (c9s) [inline .github/workflows/release.yml:129]: use `slim-dnum` in the middle?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -119,3 +119,75 @@ jobs:
...
+          asset_content_type: application/octet-stream
+      - name: Upload slim Linux arm64 dnum
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: dist/${{ steps.get_version.outputs.VERSION }}/bbgo-slim-${{ steps.get_version.outputs.VERSION }}-linux-arm64-dnum.tar.gz
+          asset_name: bbgo-slim-${{ steps.get_version.outputs.VERSION }}-linux-arm64-dnum.tar.gz
```
Comment (c9s) [inline .github/workflows/release.yml:138]: `-slim-dnum-` might be better

File: Makefile [hunk 3]
```diff
@@ -65,6 +65,48 @@ bbgo-slim-darwin-amd64: $(BIN_DIR)
...
+# build native bbgo
+bbgo-dnum: static
+	go build -tags web,release,dnum -o $(BIN_DIR)/bbgo ./cmd/bbgo
+
+# build native bbgo (slim version)
+bbgo-slim-dnum:
+	go build -tags release,dnum -o $(BIN_DIR)/$@ ./cmd/bbgo
+
+# build cross-compile linux bbgo
+bbgo-linux-dnum: bbgo-linux-amd64-dnum bbgo-linux-arm64-dnum
```
Comment (c9s) [inline Makefile:77]: `bbgo-dnum-linux`, make the platform name as the latest suffix

File: Makefile [hunk 4]
```diff
@@ -65,6 +65,48 @@ bbgo-slim-darwin-amd64: $(BIN_DIR)
...
+
+bbgo-linux-amd64-dnum: $(BIN_DIR) pkg/server/assets.go
```
Comment (c9s) [inline Makefile:79]: ditto

File: Makefile [hunk 5]
```diff
@@ -65,6 +65,48 @@ bbgo-slim-darwin-amd64: $(BIN_DIR)
...
+	GOOS=linux GOARCH=amd64 go build -tags web,release,dnum -o $(BIN_DIR)/$@ ./cmd/bbgo
+
+bbgo-linux-arm64-dnum: $(BIN_DIR) pkg/server/assets.go
```
Comment (c9s) [inline Makefile:82]: ditto

File: .github/workflows/release.yml (hunk 1 above)
Comment (zenixls2) [reply .github/workflows/release.yml]: done

File: .github/workflows/release.yml (hunk 2 above)
Comment (zenixls2) [reply .github/workflows/release.yml]: done

File: Makefile (hunk 3 above)
Comment (zenixls2) [reply Makefile]: done

File: Makefile (hunk 4 above)
Comment (zenixls2) [reply Makefile]: done

File: Makefile (hunk 5 above)
Comment (zenixls2) [reply Makefile]: done

File: scripts/setup.sh [hunk 6]
```diff
@@ -27,7 +27,7 @@ function info()
 info "downloading..."
 curl -O -L https://github.com/c9s/bbgo/releases/download/$version/$dist_file
 tar xzf $dist_file
-mv bbgo-$osf bbgo
+mv bbgo-$osf-amd64 bbgo
```
Comment (c9s) [inline scripts/setup.sh:30]: Why adding amd64 here? It could be arm64 because of Apple M1.

Comment (zenixls2) [reply scripts/setup.sh]: done

**Comment by zenixls2** [comment]
tested again based on latest main.

File: README.md [hunk 7]
```diff
@@ -77,8 +78,41 @@ bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-gri
...
+
+# grid trading strategy for max exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-grid-dnum.sh) max
+
+# bollinger grid trading strategy for binance exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-bollgrid-dnum.sh) binance
+
+# bollinger grid trading strategy for max exchange
+bash <(curl -s https://raw.githubusercontent.com/c9s/bbgo/main/scripts/setup-bollgrid-dnum.sh) max
+```
```
Comment (c9s) [inline README.md:103]: move dnum version to the advanced installation doc? we can just put the default installation in the readme.

**Comment by c9s** [comment]
We need to wait for a stable release to test this.

File: README.md (hunk 7 above)
Comment (zenixls2) [reply README.md:111]: done

**Pull request closed without merging**